catalog/registry/catalog-registry-cswinputtransformer/src/main/java/ddf/catalog/registry/transformer/RegistryTransformer.java

```diff
@@ -68,14 +57,19 @@ public Metacard transform(InputStream inputStream, String id)
             throws IOException, CatalogTransformerException {
 
         RegistryMetacardImpl metacard;
+
         String xml = IOUtils.toString(inputStream);
         IOUtils.closeQuietly(inputStream);
 
         try {
-            metacard = (RegistryMetacardImpl) xstream.fromXML(xml);
-        } catch (XStreamException e) {
+            metacard = (RegistryMetacardImpl) unmarshal(IOUtils.toInputStream(xml));
```

Comment:
We should have examples of using this in various places. Content framework comes to mind.